spec/models/order_spec.rb

```diff
@@ -68,6 +68,124 @@
     it{ is_expected.to have_db_column(:booking_type_id).of_type(:integer)}
   end
 
+  describe '.my_orders' do
```

Comment:
Good testing coverage. Might be possible to refactor to fewer lines (some tests very much look the same), although I wouldn't worry about it :)